web_translate_dialog/static/src/js/web_translate_dialog.js

```diff
@@ -61,13 +61,14 @@ var translateDialog = Dialog.extend({
             sup = this._super;
         // the template needs the languages
         $.when(this.languages_loaded).then(function() {
-            return sup.call(self);
+            return Reflect.apply(sup,self,[]);
+
```

Comment:
Why not just `return $.when(this.languages_loaded).then($.proxy(this._super, this));`?